tailwind.config.js

```diff
@@ -223,6 +223,7 @@ module.exports = {
     plugins: [
         require('tailwindcss/plugin')(({ addVariant }) => {
             addVariant('search-cancel', '&::-webkit-search-cancel-button');
+            addVariant('calendar-icon', ['&::-webkit-calendar-picker-indicator', '&::-webkit-inner-spin-button']);
```

Comment:
Don't forget the non-chromium browsers like Firefox!

Reply:
Good catch 👌🏻 It turns out you cannot style the calendar icon on Firefox (https://bugzilla.mozilla.org/show_bug.cgi?id=1812397). I updated the implementation to hide the default icon using the custom one